### DISCUSSION TITLE
GUI based staff login

### DISCUSSION THREAD

**Discussion by Skyfighter1** [Zrips/CMI#4663] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
If a staff member (or another user with a permission like cmi.login) joins he can't do anything and a GUI get oppen (player can't close it). There he have to enter his personal passwort. If he fails defined actions happens (so owners can do it theirself through commands).

### Basic explanation as to why this feature is needed and/or useful to have: ###
This would protect staff accounts and so the whole server for hackers who get acces to accounts with powerfull permissions.

**Comment by CloudeLecaw** [comment]
Login Passworts..well this just is needed for onlinemode false server, that listen to playernames and where you can copy names like u want.
For online mode true server each Passwort after the launcher is nonsense.
